charts/presto/templates/configmap-coordinator.yaml

```
@@ -29,7 +29,15 @@ data:
     discovery-server.enabled={{ or (eq .Values.mode "single") (eq .Values.mode "cluster") }}
     node-scheduler.include-coordinator={{ eq .Values.mode "single" }}
     resource-manager-enabled={{ eq .Values.mode "ha-cluster" }}
-    {{- .Values.config | nindent 4 }}
+    presto.version={{ .Values.image.tag | default .Chart.AppVersion }}{{- if .Values.velox.enabled }}wVelox{{ .Values.velox.image.tag }}{{- end }}
+    {{- if .Values.velox.enabled }}
+    native-execution-enabled=true
+    optimizer.optimize-hash-generation=false
+    regex-library=RE2J
+    use-alternative-function-signatures=true
+    experimental.table-writer-merge-operator-enabled=false
+    {{- end}}
```

Please, fix formatting
```suggestion
    {{- end }}
```